resources/migrations/000_migrations.yaml

```diff
@@ -13133,6 +13188,62 @@ databaseChangeLog:
             constraintName: unique_dimension_field_id
         # this change can roll back automatically.
 
+  - changeSet:
+      id: v46.00-032
+      author: tsmacdonald
+      comment: Added 0.46.0 -- Unique values_card
+      changes:
+        - addUniqueConstraint:
+            tableName: values_card
+            columnNames: parameterized_object_id, parameterized_object_type, parameter_id
+            constraintName: unique_parameterized_object_card_parameter
+
+  - changeSet:
+      id: v46.00-033
+      author: tsmacdonald
+      comment: Added 0.46.0 -- values_card index on connected object
+      changes:
+        - createIndex:
+            tableName: values_card
+            columns:
+              - column:
+                  name: parameterized_object_id
+            indexName: idx_values_card_parameterized_object_id
+
+  - changeSet:
+      id: v46.00-034
+      author: tsmacdonald
+      comment: Added 0.46.0 -- values_card index on connected card
+      changes:
+        - createIndex:
+            tableName: values_card
+            columns:
+              - column:
+                  name: card_id
+            indexName: idx_values_card_card_id
+
+  - changeSet:
+      id: v46.00-035
+      author: tsmacdonald
+      comment: Added 0.46.0 - values_card.card_id foreign key
+      changes:
+        - addForeignKeyConstraint:
+            baseTableName: values_card
+            baseColumnNames: card_id
+            referencedTableName: report_card
+            referencedColumnNames: id
+            constraintName: fk_values_card_ref_card_id
+            onDelete: CASCADE
+
+  - changeSet:
```

Comment:
These changes have not landed on master or anywhere else. I think we should probably just fix the migrations instead of add a migration to rename a table that we don't like the name of.

should be simple: migrate down, fix up 000_migrations.yaml, migrate up, commit

Reply:
I did it this way so that I wouldn't have to coordinate the rollback with the FE team (who are working off this branch), but I'm happy to do so

Reply:
yeah let's make it a single pass. This file grows so much that we if we don't have to do it this way we shouldn't. Just coordinate with them how to restore back to a working state.

Reply:
Please don't add hacks to migrations because of me. I'd be happy to recreate the db again if needed. I do it every several days anyway.

Reply:
done